diff --git a/drivers/pci/pci-acpi.c b/drivers/pci/pci-acpi.c
index 9cc447da9475..4d81b1a426e0 100644
--- a/drivers/pci/pci-acpi.c
+++ b/drivers/pci/pci-acpi.c
@@ -1019,8 +1019,8 @@ bool acpi_pci_bridge_d3(struct pci_dev *dev)
 		 * Otherwise, assume that the bridge can enter D3 so long as it
 		 * is power-manageable via ACPI.
 		 */
-		if (acpi_device_power_manageable(adev))
-			return true;
+		// if (acpi_device_power_manageable(adev))
+		// 	return true;
 	}
 
 	rpdev = pcie_find_root_port(dev);
